src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -47,25 +47,26 @@ public async Task<ActionResult> GetAllAsync(Guid organisationId)
 
```

Comment:
I think there's an unnecessary using in this file – could you remove it while you're here?

Reply:
👍